### DISCUSSION TITLE
Hyper V v9.0, v10.0 and VMWare 16.2 Virtual Machines images ready for use

### DISCUSSION THREAD

**Discussion by hawkeye73** [retronas/discussions#100] [Show and tell]
Hi. Just want to say WOW to Dan and Bob for this amazing project. I wanted to do something back.

https://retro.ramonddevrede.nl/retronas/

I made a HyperV v9.0 for Windows 10, HyperV v10.0 for Windows 11 and a VMWare 16.2 Virtual Machine. I couldn't upload it here for size issues so I shared it on mega. Any other service as tip is welcome. Or how to share otherwise. I hope with this people can test more and easier with those virtual machines.

My hope is there will be a docker container available in the future so I can also run RetroNAS in a docker environment.

Thanks,
Ramond/HAWKEYE73

**Comment by sairuk** [comment]
This is cool, thanks. Should help a few people out who want to get started out.
